it/v2/endpoints/SubmitForeignPropertyBsasControllerISpec.scala

```diff
@@ -672,8 +630,8 @@ class SubmitForeignPropertyBsasControllerISpec extends IntegrationBaseSpec {
           (FORBIDDEN, "ASC_ALREADY_SUPERSEDED", FORBIDDEN, RuleSummaryStatusSuperseded),
```

Comment:
I know it;'s not in yellow, but INVALID_PAYLOAD_REMOTE AND INVALID_MONETARY_FORMAT are crossed out in the spec; should probably be removed here.

I think they might have forgotten to change some parts to yellow; the mapping here doesn't match the tech spec (for example, ASC_ID_INVALID)